README.md

```diff
@@ -4,6 +4,8 @@ Run `npm install` then `npm start`, and visit `http://localhost:3000` for instru
 
 ## Submitting Your Work
 
-Please **do not submit a pull request to, or open any issues in this repo**. This is a template repository only. To submit your challenge, hit the "use this template" button on github to create a copy of it for yourself and Grant access to `kylemac`, `mwickett`, and `brkalow` as collaborators. All your work should be done on your copy of the repo only.
+Please **do not submit a pull request to, or open any issues in this repo**. This is a template repository only. To submit your challenge, hit the "use this template" button on github to create a copy of it for yourself and Grant access to `kylemac`, `mwickett`, `brkalow`, `alexcarpenter`, `ashleemboyer`, and `brandonromano` as collaborators. All your work should be done on your copy of the repo only.
```

Comment:
💭 I think updating the wording from "to submit your challenge" would be beneficial since there is a separate section with steps for submitting the challenge. I also think re-formatting this into a bullet list and providing a little more info on how to add collaborators helps make it really clear what people need to do. What do you think @mwickett?

```suggestion
Please **do not submit a pull request to, or open any issues in this repo**. This is a template repository only. All your work should be done on your copy of the repo only. To begin your challenge:

- Create a new repository using this one as a template. This can be done in two ways:
  - Clicking the "Use this template" button on the GitHub home page
  - Visiting the generate page: [mktg-webdev-challenge/generate](https://github.com/hashicorp/mktg-webdev-challenge/generate)
- In your new repository, go to Settings > Collaborators
- Add the following usernames as collaborators: `kylemac`, `mwickett`, `brkalow`, `alexcarpenter`, `ashleemboyer`, and `brandonromano`
```

Reply:
Much clearer, this is great.